### PR TITLE
Added FCSubscribe and proxy support

### PR DESCRIPTION
**Pull request by albertogasparin**
This is a modified version of [this pull request](https://github.com/videojs/video-js-swf/pull/156).
Additionally, I've added the proxy parameter in order to properly support people behind firewalls.
I assume this might solve lots of complains about "rtmp stream not playing".

Should close issue #202 

Tested with:

```
rtmp://184.72.239.149/vod/BigBuckBunny_115k.mov
rtmp://s3b78u0kbtx79q.cloudfront.net/cfx/st/honda_accord
and FB live streams (rtmps) behind a firewall
```

### CODE REVIEW

File: src/com/videojs/providers/RTMPVideoProvider.as [hunk 1]
```diff
@@ -463,6 +464,7 @@ package com.videojs.providers{
             switch(e.info.code){
                 case "NetConnection.Connect.Success":
                     _model.broadcastEventExternally(ExternalEventName.ON_RTMP_CONNECT_SUCCESS);
+                    _nc.call("FCSubscribe", null, _src.streamURL); // try to subscribe
```
Comment (gkatsev) [inline src/com/videojs/providers/RTMPVideoProvider.as]: would this cause any issues for normal playback like for mp4s?

Comment (albertogasparin) [inline src/com/videojs/providers/RTMPVideoProvider.as]: I've tested it with (node-rtsp-rtmp-server)[https://github.com/iizukanao/node-rtsp-rtmp-server], as it does not support FCSubscribe. VOD videos play fine: as the streaming server reports "unknown command" the swf code ignores it.

Comment (ashiguruma) [inline src/com/videojs/providers/RTMPVideoProvider.as]: I can second this. On-demand videos play fine when using the updated SWF.

**Comment by cat2608** [comment]
This change is working for me too. Could you please review and merge? Thanks!!

**Comment by albertogasparin** [comment]
Bump. Can we get this sorted?

**Pull request closed without merging**